package.json

```diff
@@ -259,6 +259,10 @@
         {
           "command": "extension.disconnectObjectExplorerNode",
           "when": "viewItem != null"
+        },
+        {
+          "command": "extension.toggleSqlCmd",
+          "when": "editorFocus"
         }
```

Comment:
Shouldn't this also only be available when the editor is connected? What happens currently if you do it for a disconnected action?

Reply:
It will prompt for a connection and run the query if not connected or just run

Reply:
Awesome. And it'll correctly be in SQLCMD mode for that query that's ran?

Reply:
Yes

Reply:
@Charles-Gagnon Actually, the behavior now is even better. SQL CMD is a switch that can be on or off when editing a sql file, and that option is respected for that file until a connection is made/query is run.